README.md

```diff
@@ -17,6 +17,32 @@ between two git revisions.
 
 It will configure `@owner` as the default owner in the config file.
 
+
+### Generating and validating owners
+
+By default, owners will be validated either against prepopulated file (teams.txt) or against data fetched from github.
+Format of teams.txt is one owner per line.
+
+Example teams.txt:
```

Comment:
I know I suggested the file name, now that I think of it `owners.txt` sounds better.